Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
@lilleyse do you recall:

> The Cesium API should not expose each point as a feature; the overhead is too high

from https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/22#issuecomment-231777777.

I suppose now is the time to think this through.

My thought here was that per-point `Cesium3DTileFeature` is just a incredible amount of memory overhead.  Instead, per-point batch table values would be stored in batch table textures and accessed in the shader for exploration and visualization (e.g., "show only points with some property > 0.5" or "map this property to this color ramp.").

The downside here is that the user can't inspect and change the per-point properties in JavaScript.  Perhaps the best option is to make this an option that defaults to `false`?